odl/operator/default_ops.py

```diff
@@ -103,6 +104,25 @@ def inverse(self):
     def adjoint(self):
         """Adjoint, given as scaling with the conjugate of the scalar.
 
+        Examples
+        --------
+        In the real case, the adjoint is the same as the operator
+
+        >>> r3 = odl.rn(3)
+        >>> x = r3.element([1, 2, 3])
+        >>> op = ScalingOperator(r3, 2)
+        >>> op.adjoint(x) == op(x)
+        True
+
+        In the complex case, the scalar is conjugated
+
+        >>> c3 = odl.cn(3)
+        >>> x_complex = c3.element([1, 1j, 1-1j])
+        >>> op = ScalingOperator(c3, 1+1j)
+        >>> expected_op = ScalingOperator(c3, 1-1j)
+        >>> op.adjoint(x_complex) == expected_op(x_complex)
```

Comment:
I'd just show the two side by side. This is more unit-test-like.

Reply:
How do I show them side by side? First one of them, with expected result underneath, and then the other?

I guess I could do the same just above.

Reply:
Exactly. With "side" I meant "top/bottom" :wink: